cloud/ec2_client.go

```diff
@@ -155,6 +155,7 @@ type generateDeviceNameOptions struct {
 const (
 	awsClientImplRetries     = 10
 	awsClientImplStartPeriod = time.Second
+	awsClientMaxBackoff      = 10 * time.Second
```

Comment:
Can we make this higher and reduce the number of retries as well? If it's used for every single call, the exponential backoff means that it'll try for 1, 2, 4, 8 seconds and then it won't be exponential backoff beyond 4 retries. They might rate-limit us if we don't back off enough.

Reply:
Should we just get rid of the max backoff then and reduce the number of retries?

Reply:
We resolved this by just reducing the number of retries and not imposing a global per-attempt timeout on every single call.